source/tools/index.rst

```diff
@@ -141,10 +147,10 @@ Following operations are supported::
       #
       #    Name ,  ImageFile      ,CompAlg  ,  AuthType,       KeyFile                 , Alignment,  Size
       # ===================================================================================================
-        ( 'BOOT', 'Out'           , ''      , 'RSA2048', 'TestSigningPrivateKey.pem'   ,  0x10,       0),  <--- Container Hdr
-        ( 'CMDL', 'cmdline.txt'   , 'Lz4'   , 'RSA2048', 'TestSigningPrivateKey.pem'   ,  0,          0),  <--- Component Entry 1
-        ( 'KRNL', 'vmlinuz'       , 'Lz4'   , 'RSA2048', 'TestSigningPrivateKey.pem'   ,  0,          0),  <--- Component Entry 2
-        ( 'INRD', 'initrd'        , 'Lz4'   , 'RSA2048', 'TestSigningPrivateKey.pem'   ,  0x1000,     0),  <--- Component Entry 3
+        ( 'BOOT', 'Out'           , ''      , 'RSA2048_SHA2_256', 'TestSigningPrivateKey.pem'   ,  0x10,       0),  <--- Container Hdr

```

Comment:
For this example, let us use some standard layout container.  Not mono-signing format.

Reply:
Updated.